test/torch/tensors/test_replicated_shared.py

```diff
@@ -180,3 +180,142 @@ def test_workers_arrangement(workers):
     me, bob, alice = (workers["me"], workers["bob"], workers["alice"])
     x = torch.tensor(7).share(bob, alice, me, protocol="falcon")
     assert x.players[0] == me
+
+
+def test_fixed_precision_and_sharing(workers):
```

Comment:
fixed pointed arithmetic is not test thought it's the purpose of that issue, numbers after decimal point should

Reply:
fixed